swmm-toolkit/src/swmm/toolkit/solver.i

```diff
@@ -209,7 +220,10 @@
 %ignore swmm_getObjectIndex;
 %ignore swmm_freeMemory;
 
+
+#define EXPORT_TOOLKIT
```

Comment:
Move to src/swmm/toolkit/CMakeLists.txt 
```
set_target_properties(solver
    PROPERTIES
        SUFFIX ${PYTHON_SUFFIX}
        SWIG_COMPILE_DEFINITIONS EXPORT_TOOLKIT
        MACOSX_RPATH TRUE
        SKIP_BUILD_RPATH FALSE
        BUILD_WITH_INSTALL_RPATH FALSE
        INSTALL_RPATH "${PACKAGE_RPATH}"
        INSTALL_RPATH_USE_LINK_PATH TRUE
)
```

Reply:
excellent choice, I was kind of stumbling around swig stuff. Thanks!